kandroid/src/main/kotlin/com/pawegio/kandroid/KDialog.kt

```diff
@@ -149,101 +148,67 @@ class KAlertDialogBuilder(val ctx: Context) {
     val builder: AlertDialog.Builder = AlertDialog.Builder(ctx)
     var dialog: AlertDialog? = null
 
-    fun dismiss() {
-        dialog?.dismiss()
-    }
+    fun dismiss() = dialog?.dismiss()
 
     fun show(): KAlertDialogBuilder {
         dialog = builder.create()
         dialog!!.show()
         return this
     }
 
-    fun title(title: CharSequence) {
-        builder.setTitle(title)
-    }
+    fun title(title: CharSequence) = builder.setTitle(title)
```

Comment:
`title()` would return `AlertDialog.Builder` instead of `Unit` here. Same below.